src/app/api/users/route.ts

```
@@ -1,5 +1,28 @@
 import { prisma } from "@/lib/db";
 import { NextResponse } from "next/server";
+import { withPermission } from "@/middleware/check-permission";
+import { PERMISSIONS } from "@/lib/constants/roles";
+import { PrismaClientKnownRequestError } from '@prisma/client/runtime/library';
+
+// Utility to robustly extract error code from error objects, even if nested
+function getErrorCode(error: any): string | undefined {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)